src/scripts/modules/media/body/content-common.less

```diff
@@ -346,16 +346,6 @@ table {
   background-color: @table-bg;
   counter-increment: table;
 
-  ul {
```

Comment:
I am concerned with regression testing if we modify `content-common.less` since this affects every single book that isn't baked (that is a vast majority of our content). I think it might be wiser to overwrite it in `baked-content.less` to avoid a huge regression testing situation.

Reply:
@helenemccarron I added this code earlier and you advise me to move it to content.baked so I did but I removed only one part and left behind that part

Reply:
Oh, that was added in a different PR? Can you please link it here since it's not obvious in this PR that we are removing something we previously added.

Reply:
I added this here `https://github.com/Connexions/webview/pull/1745/commits` and then moved this `https://github.com/Connexions/webview/pull/1786/commits/34c68fe9557bd41e07374207b7fc586fc00c8732` but content is still there